charts/openmetadata/values.yaml

```
@@ -5,6 +5,9 @@ replicaCount: 1
 
 # Overrides the openmetadata config file with the help of Environment Variables
 # Below are defaults as per openmetadata-dependencies Helm Chart Values
+labels: {}
+anootations: {}
```

there is a typo here - 
```suggestion
annotations: {}
```

---

modified annotations spelling